cmd/lakefs/cmd/migrate.go

```diff
@@ -18,32 +20,49 @@ var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Print current migration version and available version",
 	Run: func(cmd *cobra.Command, args []string) {
-		logger := logging.Default()
 		cfg := loadConfig()
-
-		ctx := cmd.Context()
 		kvParams := cfg.GetKVParams()
+		ctx := cmd.Context()
 		kvStore, err := kv.Open(ctx, kvParams)
 		if err != nil {
-			logger.WithError(err).Fatal("Failed to open KV store")
+			_, _ = fmt.Fprintf(os.Stderr, "Failed to open KV store: %s\n", err)
+			os.Exit(1)
 		}
 		defer kvStore.Close()
 
-		version, err := kv.GetDBSchemaVersion(ctx, kvStore)
-		if err != nil {
-			logger.WithError(err).Fatal("Failed to get DB schema version")
-			return
-
-		}
-		logger.WithField("schema_version", version).Info("DB schema version")
+		version := MustValidateSchemaVersion(ctx, kvStore)
 		fmt.Printf("Database schema version: %d\n", version)
 	},
 }
 
+func MustValidateSchemaVersion(ctx context.Context, kvStore kv.Store) int {
```

Comment:
Does it need to be public?

Reply:
No - its my default. No one imports cmd package.

Reply:
Updated to be private